lib/turn.rb

```diff
@@ -1,32 +1,52 @@
 class Turn
-  attr_reader :player, :board
+  attr_reader :player, :board, :columns
 
   def initialize(player, board)
     @player = player
     @board = board
+    @columns = ["a", "b", "c", "d", "e", "f", "g"]
```

Comment:
Yeah good call on committing to this... we were using columns all over the place